### PR TITLE
[RTC-364] Revisit documentation

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

File: docs/introduction/what_is_jellyfish.md [hunk 1]
```diff
@@ -14,14 +14,18 @@ In general, there are no limitations.
...
-* client and server SDKs
-* support for multiple media protocols
-* docker images for multiple architectures
+* [built-in authentication](../getting_started/authentication.md)
+* [client and server SDKs](../getting_started/sdks.md)
+* [server side notifications](../getting_started/notifications.md)
+* [support for multiple media protocols](#supported-protocols)
+* [built-in clustering](../cluster.md)
+* [tutorials](../tutorials/dashboard.mdx)
+* [docker images for multiple architectures](https://github.com/jellyfish-dev/jellyfish/pkgs/container/jellyfish)
```
Comment (sgfn) [inline docs/introduction/what_is_jellyfish.md:23]:
```suggestion
* [server-side notifications](../getting_started/notifications.md)
* [support for multiple media protocols](#supported-protocols)
* [built-in clustering](../cluster.md)
* [tutorials](../tutorials/dashboard.mdx)
* [Docker images for multiple architectures](https://github.com/jellyfish-dev/jellyfish/pkgs/container/jellyfish)
```

File: docs/introduction/what_is_jellyfish.md [hunk 2]
```diff
@@ -14,14 +14,18 @@ In general, there are no limitations.
...
-* built-in authentication
-* client and server SDKs
-* support for multiple media protocols
-* docker images for multiple architectures
+* [built-in authentication](../getting_started/authentication.md)
+* [client and server SDKs](../getting_started/sdks.md)
+* [server side notifications](../getting_started/notifications.md)
+* [support for multiple media protocols](#supported-protocols)
+* [built-in clustering](../cluster.md)
+* [tutorials](../tutorials/dashboard.mdx)
```
Comment (Rados13) [inline docs/introduction/what_is_jellyfish.md:22]: Shouldn't we have a page with `tutorials` where we would describe all available tutorials?

File: docs/tutorials/dashboard.mdx [hunk 3]
```diff
@@ -43,7 +43,7 @@ We started the dashboard and the Jellyfish Media Server separately, so we need t
   but it is meant to authenticate the dashboard to the Jellyfish, so the good practice is to use a token that is hard to guess.
 - **WS/WSS** - whether to use a secure or insecure connection. , it's `ws`, but if the Jellyfish uses secure WebSocket, use `wss`.
 - **HTTP/HTTPS** - whether to use a secure or insecure connection. By default, it's `http`, but if the Jellyfish uses secure HTTP, use `https`.
-- **Server URL** - URL of the Jellyfish server. Here, `localhost:5002` would be a local build, but it can be any URL that points to the Jellyfish.
+- **Server URL** - URL of the Jellyfish server. Here, `localhost:4000` would be a local build, but it can be any URL that points to the Jellyfish.
```
Comment (sgfn) [inline docs/tutorials/dashboard.mdx:46]: Why this change?

**Review by sgfn — APPROVED** [review]

File: docs/getting_started/metrics.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Metrics
 
-Jellyfish exports several metrics in the Prometheus format. They are available on port `9568` at `/metrics`.
+Jellyfish exports several metrics in the Prometheus format.
+By default, they are available on port `9568` at `/metrics`.
+See `JF_METRICS_IP` and `JF_METRICS_PORT` for configuration options.
```
Comment (Rados13) [inline docs/getting_started/metrics.md:5]: Shouldn't we also say here about BEAM metrics available?

**Review by Rados13 — APPROVED** [review]

File: docs/tutorials/dashboard.mdx (hunk 3 above)
Comment (mickel8) [reply docs/tutorials/dashboard.mdx:46]: That's how Jellyfish is run in the tutorial. We export port 4000

File: docs/introduction/what_is_jellyfish.md (hunk 2 above)
Comment (mickel8) [reply docs/introduction/what_is_jellyfish.md:22]: I don't think so. They are listed when someone goes to the `Tutorials` section. We would have to duplicate content
